packages/electrode-redux-router-engine/lib/redux-router-engine.js

```diff
@@ -56,7 +56,11 @@ class ReduxRouterEngine {
             `redux-router-engine: ${location} doesn't allow request method ${req.method}`);
         }
 
-        return this._handleRender(req, match, options || {});
+        try {
+          return this._handleRender(req, match, options || {})
+        } catch(err) {
```

Comment:
this promise error catch would/should be handled here https://github.com/electrode-io/electrode/pull/81/files#diff-86fb219987335569ce41401851671a43R65

unless something funky is happening inside of `_handleRender`